remove the term "active path" as it was not well defined (#435)
quicwg/multipath: draft-ietf-quic-multipath.md

(see {{path-abandon-frame}}),
- PATH_BACKUP and PATH_AVAILABLE frames to express a preference
in path usage (see {{path-backup-frame}} and {{path-available-frame}}), and

> - MAX_PATH_ID frame for increasing the limit of open paths.

Reviewer: Just here that I think the "open" word is a bit misleading (but original text was probably)
```suggestion
- MAX_PATH_ID frame for increasing the limit of usable path identifiers.
```

- PATH_BACKUP and PATH_AVAILABLE frames to express a preference
in path usage (see {{path-backup-frame}} and {{path-available-frame}}), and
- MAX_PATH_ID frame (see {{max-paths-frame}}) for increasing the limit of

> open paths, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})

Author: ```suggestion
path identifiers, and PATHS_BLOCKED frame (see {{paths-blocked-frame}})
```

PATH_BACKUP marks a path as "standby", i.e., it suggests that no traffic
should be sent on that path if another path is available.

> If all established paths are marked as "standby", no guidance is provided about

Author: use "open" here as well?

The QUIC specification defines an optional keep alive process, see {{Section 5.3 of QUIC-TRANSPORT}}.
Implementations of the multipath extension should map this keep alive process to a number of paths.

> Some applications may wish to ensure that one only path remains usable, while others could prefer to maintain

Author: ```suggestion
Some applications may keep only one path alive, while others could prefer to maintain
```

However, while {{QUIC-TRANSPORT}} only allows the use of one path simultaneously
and therefore only one path migration at the time should be validated,

> this extension allows for multiple establihed paths, that could in theory be migrated

Author: ```suggestion
this extension allows for multiple open paths, that could in theory be migrated
```
